governance/Community_spaces.md

```diff
@@ -0,0 +1,30 @@
+# UK TRE Community working and communication spaces
+
+## Jiscmail: Open discussions and annoucements
+
+Our mailing list (UK-TRE-COMM@JISCMAIL.AC.UK) is the main and most important space to watch out for announcements.
+All official communications are made here, such as events.
+
+The list is a Public Discussion and everyone (even non subscribers) can send messages, all messages are public and available [here](https://www.jiscmail.ac.uk/cgi-bin/webadmin?A0=uk-tre-comm).
+The use of the list should adhere to our [Code of Conduct](https://github.com/uk-tre/website/blob/615735dfe9d428160be3de8848a7f71d43918d90/CODE_OF_CONDUCT.md) as well as JISC [Service Policies](https://www.jiscmail.ac.uk/policyandsecurity/).
+The list is currently unmoderated, but we ask that everyone keeps their messages within topic.
+
+We encourage community members unsure about the suitability of their communications to contact a member of the Community Management Working Group, one of its chairs or to write to our inbox at uktrecommunity@gmail.com
+
+## Slack 
+We've moved! We are no longer maintaining the previous slack channel within the RSE workspace
+
+uktrecommunity.slack.com is our NEW slack workspace, it is intended as a place to work towards the outputs of specific working groups.
+Currently is where the Community Management Working Group coordinates, it is also used by chairs to discuss privately (on the rare ocassions that is needed, like sharing personal details).
+There are no specific channels, but these will be created as and if required by emerging interest or working groups.
+
+While everyone is welcome to make an announcement here it is NOT the expected place for them.
+
+## Github: Org, repositories and project tracker
+
+< general overview and reference to future specific doc on how to use the repo, or just do it here?>
+
+## NEW: proposed announcement list
```

Comment:
I think this doesn't need to be here - maybe captured as an issue on `Backlog`?

Reply:
Answering on issue #40